ulib/FStar.Algebra.CommMonoid.Fold.fst

```diff
@@ -114,3 +117,39 @@ let rec fold_equals_seq_foldm #c #eq
     eq.symmetry rhs (subfold `op` last);
     eq.transitivity lhs (subfold `op` last) rhs
   )
+ 
```

Comment:
I should have requested an interface for this file too when merging a previous PR. Could you please add one?

Reply:
No problem, done :)